Header header
float64 translation
float64 rot1
float64 rot2
string state